OSDOCS-9541 OCP z-stream RNs 4.14.11 (#71094)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.14.11 --pullspecs
----

[id="ocp-4-14-11-features"]
==== Features

The following feature is included in this z-stream release:

[id="ocp-4-14-11-wherabouts-cron-schedule"]
===== Enabling configuration of whereabouts cron schedule

* The Whereabouts reconciliation schedule was hard-coded to run once per day and could not be reconfigured. With this release, a ConfigMap has enabled the configuration of the whereabouts cron schedule. xref:../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-creating-whereabouts-reconciler-daemon-set_configuring-additional-network[Creating a Whereabouts reconciler daemon set].

[id="ocp-4-14-11-bug-fixes"]
==== Bug fixes

> * Previously, upgrading {product-title} can lead to DNS quieries failing due to upstream returning a payload larger than 512 bytes for non-EDNS queries using CoreDNS 1.10.1. With this release, clusters with a non-compliant upstream will retry with TCP upon overflow errors which will prevent disruption of function when upgrading. (link:https://issues.redhat.com/browse/OCPBUGS-28200[*OCPBUGS-28200*])

Author: Changed.

> * Previously, upgrading {product-title} can lead to DNS quieries failing due to upstream returning a payload larger than 512 bytes for non-EDNS queries using CoreDNS 1.10.1. With this release, clusters with a non-compliant upstream will retry with TCP upon overflow errors which will prevent disruption of function when upgrading. (link:https://issues.redhat.com/browse/OCPBUGS-28200[*OCPBUGS-28200*])

Author: Changed.

> * Previously, the node.env file would be overwritten on every restart due to a typo in an environment variable. With this release, edits to the node.env will persist after a restart. (link:https://issues.redhat.com/browse/OCPBUGS-27362[*OCPBUGS-27362*])

Reviewer: Per [ISG](https://www.ibm.com/docs/en/ibm-style?topic=elements-files-directories) I believe files should be in monospace font:
```suggestion
* Previously, the `node.env` file would be overwritten on every restart due to a typo in an environment variable. With this release, edits to the `node.env` will persist after a restart. (link:https://issues.redhat.com/browse/OCPBUGS-27362[*OCPBUGS-27362*])
```

> * Previously, `container_t` could not access DRI (Direct Rendering Infrastructure) devices. With this release, the policy has been updated so `container_t` can now access DRI devices and GPU devices exposed by a device plugin by default. (link:https://issues.redhat.com/browse/OCPBUGS-27275[*OCPBUGS-27275*])

Reviewer: I get the use of the parentheses to clarify what's probably more often used as an acronym, but the parentheses should probably be used to introduce the acronym for the full term:
```suggestion
* Previously, `container_t` could not access Direct Rendering Infrastructure (DRI) devices. With this release, the policy has been updated so `container_t` can now access DRI devices and GPU devices exposed by a device plugin by default. (link:https://issues.redhat.com/browse/OCPBUGS-27275[*OCPBUGS-27275*])
```

Author: You are correct, thanks!

> * Previously, `container_t` could not access DRI (Direct Rendering Infrastructure) devices. With this release, the policy has been updated so `container_t` can now access DRI devices and GPU devices exposed by a device plugin by default. (link:https://issues.redhat.com/browse/OCPBUGS-27275[*OCPBUGS-27275*])

Reviewer: Also, per [ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#p) the word "plug-in" in is preferred:
```suggestion
* Previously, `container_t` could not access DRI (Direct Rendering Infrastructure) devices. With this release, the policy has been updated so `container_t` can now access DRI devices and GPU devices exposed by a device plug-in by default. (link:https://issues.redhat.com/browse/OCPBUGS-27275[*OCPBUGS-27275*])
```

Author: Changed.

> * Previously, pods assigned an IP from the pool created by the Whereabouts CNI plugin were getting stuck in `ContainerCreating` state after a node force reboot. With this release, the Whereabouts CNI plugin issue associated with the IP allocation after a node force reboot is resolved.(link:https://issues.redhat.com/browse/OCPBUGS-26553[*OCPBUGS-26553*])

Reviewer: Up to you but adding "address" would be more explicit and clear imo:
```suggestion
* Previously, pods assigned an IP address from the pool created by the Whereabouts CNI plugin were getting stuck in `ContainerCreating` state after a node force reboot. With this release, the Whereabouts CNI plugin issue associated with the IP allocation after a node force reboot is resolved.(link:https://issues.redhat.com/browse/OCPBUGS-26553[*OCPBUGS-26553*])
```

Author: Added.

> * Previously, pods assigned an IP from the pool created by the Whereabouts CNI plugin were getting stuck in `ContainerCreating` state after a node force reboot. With this release, the Whereabouts CNI plugin issue associated with the IP allocation after a node force reboot is resolved.(link:https://issues.redhat.com/browse/OCPBUGS-26553[*OCPBUGS-26553*])

Reviewer: Ditto re: plug-in:
```suggestion
* Previously, pods assigned an IP from the pool created by the Whereabouts CNI plug-in were getting stuck in `ContainerCreating` state after a node force reboot. With this release, the Whereabouts CNI plug-in issue associated with the IP allocation after a node force reboot is resolved.(link:https://issues.redhat.com/browse/OCPBUGS-26553[*OCPBUGS-26553*])
```

Author: Changed, thank you!
